### PR TITLE
Assessment 9111

### PR DESCRIPTION
**Pull request by shahidrafique**
https://edx-wiki.atlassian.net/browse/MCKIN-9111
Ensure Assessments are responsive in a web browser on all devices and applies New UI styles.

Add styling according to the new theme. work for responsive devices as well.

### CODE REVIEW

File: lms/static/js/apros-xblocks.js [hunk 1]
```diff
@@ -30,9 +30,27 @@ $(function () {
...
+    $(".new-theme .choices-list input[type=radio]:checked, .new-theme input[type=radio]:checked").parent().addClass('selected');
+  });
+
+// Assessment block checkbox
+
+$(document).on('DOMNodeInserted', '.choices-list, .choice-selector', function(){
+  $(".new-theme .choice-selector input[type=checkbox]:checked").parent().addClass('selected');
+});
+
+
```
Comment (msaqib52) [inline lms/static/js/apros-xblocks.js]: nit: please remove extra empty lines

File: lms/static/js/apros-xblocks.js [hunk 2]
```diff
@@ -30,9 +30,27 @@ $(function () {
...
+
+
   $(document).on('DOMNodeInserted', '.forum-new-post-form', function(){
     $(".new-theme input[type=checkbox]:checked").parent().addClass('selected');
   });
+
+
```
Comment (msaqib52) [inline lms/static/js/apros-xblocks.js]: nit: please remove extra empty lines

File: lms/static/js/apros-xblocks.js [hunk 3]
```diff
@@ -30,9 +30,27 @@ $(function () {
...
   });
+
+// assessment block choice
+  $(document).on('DOMNodeInserted', '.choices-list, .choice, .choice-selector', function(){
+    $(".new-theme .choices-list input[type=radio]:checked, .new-theme input[type=radio]:checked").parent().addClass('selected');
+  });
+
+// Assessment block checkbox
+
+$(document).on('DOMNodeInserted', '.choices-list, .choice-selector', function(){
```
Comment (msaqib52) [inline lms/static/js/apros-xblocks.js]: nit: please indent appropriately.

File: lms/static/sass/xblocks/assessment.scss [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+@import 'core/core';
+@import 'form/checkbox';
+@import 'form/radiobutton';
+@import 'common/buttons';
+
+
+
+.new-theme
+{
+    [data-block-type="step-builder"]
```
Comment (hussnain2) [inline lms/static/sass/xblocks/assessment.scss]: @shahidrafique Please Do code formatting on all SCSS files.

Comment (shahidrafique) [reply lms/static/sass/xblocks/assessment.scss]: this is individual SCSS for xBlock.

**Pull request merged**